cmd/ttn-lw-cli/internal/util/flags.go

```diff
@@ -581,7 +602,7 @@ func setField(rv reflect.Value, path []string, v reflect.Value) error {
 							break
 						}
 						var enum ttnpb.RxDelay
-						if err := enum.UnmarshalText([]byte(v.String())); err != nil {
+						if err := enum.UnmarshalText([]byte(v.String())); err != nil && enum != 0 {
```

Comment:
Technically this can also be set to `0`, however we probably don't want to allow that.
This does not belong here though, and, if set to `0` CLI should ideally silently convert it to `1`, but still set. I think the simplest solution here is to just allow `0` for now and later think about conversion.

Reply:
I agree with your comment, but I also do not think this is relevant to this PR. Resolving conversation just like the others.